src/controller/SDLController.cpp

```diff
@@ -241,12 +211,6 @@ void SDLController::ReadFromSource(int32_t virtualSlot) {
                     SPDLOG_TRACE("Invalid PosStickX configured. Neg was {} and Pos is {}", leftStickAxisX, axis);
                 }
 
-                if (leftStickDeadzone != 0 && leftStickDeadzone != axisDeadzone) {
```

Comment:
Why remove all the warnings about the deadzones being different before adding code to process the x/y deadzones separately?

Reply:
No, it's ok to keep them removed for now. I'll add a comment for the change we can make for now, even if it's unused.